c2corg_ui/static/js/maincontroller.js

```diff
@@ -76,3 +91,17 @@ app.MainController.prototype.isPath = function(path) {
 app.MainController.prototype.animateHeaderIcon = function(e) {
   app.utils.animateHeaderIcon(e);
 }
+
+
+/**
+ * @export
+ */
+app.MainController.prototype.redirect = function(url) {
```

Comment:
It's not obvious that the function is used for protected URLs which require to be authenticated. You don't want to use it for public URLs.
Maybe ``redirectToUrlWhichRequiresToBeAuthenticated`` :P  or ``checkAuthAndRedirect`` or ```redirectToProtectedUrl``?